File: challenges/1-media-alunos/c/spacexjedi/media-alunos.c
#include <stdio.h>

float media_aluno(float n1, float n2) {

  float media = (n1 + n2)/2;

  return media;
}
   

int main(void) {

  float media = media_aluno(6, 4); // passe os valores desejados para a função
  
  printf("%.2f", media);

  return 0;
}

File: challenges/1-media-alunos/java/tarcila414/MediaAlunos.java
public class MediaAlunos {

  /**
   * Função para calcular a média aritmética entre dois valores
   * 
   * @param A
   * @param B
   * @return
   */
  public static double calcularMedia(double A, double B) {
    return (double) ((A + B) / 2);
  }

  public static void main(String[] args) {
    try {
      if (args != null && args.length >= 2) {
        double media = calcularMedia(Double.parseDouble(args[0]), Double.parseDouble(args[1]));
        System.out.printf("Média -> %.2f \n", media);
      }
    } catch (NumberFormatException e) {
      System.out.println("ERRO => Você deve inserir dois números no seguinte formato: xx.xx ou x");
    }
  }
}